arches/app/media/js/views/graph/graph-designer/card-tree.js

```diff
@@ -370,7 +372,21 @@ define([
                 });
             },
             selection: selection,
-            filter: filter
+            filter: filter,
+            isFuncNode: function() {
+                var appFuncDesc, appFuncName, nodegroupId;
+                if(params.card) {
+                    nodegroupId = params.card.nodegroup_id;
+                    if(this.appliedFunctions().length > 0) {
+                        appFuncDesc = this.appliedFunctions()[0]['config']['description']['nodegroup_id'];
```

Comment:
this section has the same problems as described above re: `isFuncNode` in card.js,  ie keying on properties that may not be present and assuming the function of interest is at index 0 of the array.